src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExceptionsPipelineTests.cs

```diff
@@ -53,7 +53,7 @@ public Task EventExceptionsPipeline_SingleException()
                     Assert.False(string.IsNullOrEmpty(instance.Message));
```

Comment:
I added an extra test for this - if we'd rather, I can also convert an existing test to have this parameter, since it's currently effectively a duplicate of the `CustomException` test.

Reply:
Please simplify that test so it's sole focus is on the call stack e.g. throw a built in exception without an inner exception and rename the test to something like EsotericStackFrameTypes.

Also, consider adding `unsafe` types as one of the parameters e.g. `void*` to flex more of the special cases that are less commonly encountered.